extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/guice/MSQDurableStorageModule.java

```diff
@@ -47,6 +51,7 @@ public class MSQDurableStorageModule implements DruidModule
   @Inject
   private Properties properties;
 
+  // Dummy constructor so that it can get injected dynamically at runtime
```

Comment:
does moving `@Inject` from member variable `properties` to `MSQDurableStorageModule(Properties properties)` constructor also work, which could make the default constructor obsolete?

Reply:
I don't have an idea of the same, but I will check. Thanks for the suggestion.

Reply:
I tried the suggestion and it didn't work.

Reply:
discussed offline - removed constructors and moved the properties injection to a separate method like other extensions. thanks for the changes!